addons/mail/static/tests/activity/activity.test.js

```diff
@@ -404,6 +404,41 @@ test("activity click on edit", async () => {
     await assertSteps(["do_action"]);
 });
 
+QUnit.test("activity click on edit should pass correct context", async (assert) => {
```

Comment:
Remove `assert` in the params. This is not how we assert stuffs in HOOT. I will give example for how to convert the assertion statements.